src/main/java/by/itech/lab/supplier/dto/ApplicationDto.java

```diff
@@ -0,0 +1,28 @@
+package by.itech.lab.supplier.dto;
+
+import by.itech.lab.supplier.domain.ApplicationStatus;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+
+@Builder
+@Data
+public class ApplicationDto implements BaseDto {
+
+    Set<ItemsInApplicationDto> items = new HashSet<>();
+    private Long id;
+    private String number;
+    private LocalDate registrationDate;
+    private LocalDate lastUpdated;
+    private AddressDto sourceLocationAddressIdDto;
```

Comment:
Could be named as sourceLocationAddressDto

Reply:
+++